builder/docker/docker_test.go

```diff
@@ -0,0 +1 @@
+package docker
```

Comment:
If this file amounts to just a placeholder for tests, maybe it shouldn't be included. When I scan a go package and see foo_test.go, my assumption is "good, that means foo.go has tests."